bindings/rust/evmc-vm/src/container.rs

```diff
@@ -2,10 +2,10 @@
 // Copyright 2019 The EVMC Authors.
 // Licensed under the Apache License, Version 2.0.
 
-use crate::EvmcVm;
-
 use std::ops::{Deref, DerefMut};
 
+use crate::EvmcVm;
```

Comment:
Does moving this line of code have any effect? 

If not, I would like to see it undone to keep the diff from the base evmc version as small as possible. We will have to eventually rebase the changes of this repo to up-stream changes, and the less modifications there are, the easier this will be.